site/src/atomics/background.md

```diff
@@ -0,0 +1,64 @@
+| -------------------------- | ----------------------------------------------- | ---------- |
+| `background-size`          | `100`, `200`                                    | n/a        |
+| `background-image-pattern` | Applies `var(--background-image-pattern)` value | n/a        |
+
+## Usage
+
+First in order to apply a background pattern, specify the `--background-image-pattern` variable in a style tag. Then include the `background-image-pattern` class. Note: if you intend to apply a background color as well, you should ensure your asset is transparent. You can do this by authoring a transparent png, or by adding [`fill-opacity`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-opacity) (try `.1`) to an existing opaque svg. You may also combine this with background color classes from color atomics.
+
+```html
+<section
+	class="padding-lg background-image-pattern background-repeat-repeat background-color-primary color-primary-invert"
+	style="--background-image-pattern: url('https://docs.microsoft.com/en-us/media/background-patterns/pixie-sticks.svg')"
+>
+	<h2 class="font-size-h2">A confetti background</h2>
+	<p>with <code>fill-opacity=".1"</code> on an svg</p>
+</section>
+```
+
+## Adjusting your repeat settings with background sizes
+
+You can adjust the look and feel of your pattern with different `background-size` values.
+
+```html
+<section
+	class="background-size-100 background-image-pattern padding-lg background-color-primary color-primary-invert"
+	style="--background-image-pattern: url('https://docs.microsoft.com/en-us/media/background-patterns/plus.svg')"
+>
+	<p><code>background-size-100</code></p>
+</section>
+<section
+	class="background-size-200 background-image-pattern padding-lg margin-top-xxs background-color-primary color-primary-invert"
+	style="--background-image-pattern: url('https://docs.microsoft.com/en-us/media/background-patterns/plus.svg')"
+>
+	<p><code>background-size-200</code></p>
+</section>
+```
+
+## Theming and accessibility concerns
+
+Be aware that a transparent svg will not be themeable, so while background colors will differ on different themes, your svg or png will not. With semi-transparent greyscale images, this can still look good, but you should ensure you test carefully on different themes to achieve the result you're looking for. Additionally, if your image is not made transparent enough, it may negatively affect the readability (contrast ratio) of text on your background.
+
+```html
+<section
+	class="padding-lg background-color-primary color-primary-invert background-image-pattern background-repeat-repeat background-size-100"
+	style="--background-image-pattern: url('https://docs.microsoft.com/en-us/media/background-patterns/plus.svg')"
+>
+	<p><code>background-size-100</code></p>
+</section>
+```
+
+## Why don't we just inline these SVGs into Atlas?
+
+Keeping svg source code out of Atlas reduces our file sizes, and maintains flexibility for you the developer.
```

Comment:
I've been forgetting about this, but should we add a `Available Classes` list? Right now, some atomic pages has it, while others don't.

Reply:
If I'm not mistaken it should be added if there are quite a lot of atomics, don't remember the exact number though.

Reply:
Added to this page at least

Reply:
That's why it wasn't added to all of the pages, causes weird spacing on the bootom:

![image](https://user-images.githubusercontent.com/57374379/184717686-e9794a3e-9e1c-4715-a881-1284201cc7b8.png)